### PR TITLE
Added i18n

### PR DESCRIPTION
**Pull request by priscilawebdev**
Type: Feat

Description: "UI i18 - Internationalization "
Task: #1065

* Done:
 - [x] Added React-Intl and babel-plugin-react-intl
 - [x] Added configurations
 - [x] Added translations for Header and Footer

* In Progress:
 - [ ] **Tests (Attention please)**
 - [ ]  **Define how the JSON translation will be handle**
 - [ ]  **Bundle size**

Please leave your opinion / suggestion below. Really appreciate!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +57,7 @@
     "@commitlint/config-conventional": "7.1.2",
     "@material-ui/core": "3.1.0",
     "@material-ui/icons": "3.0.1",
+    "@types/react-intl": "2.3.11",
```
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply package.json]: Hi! Please it is in progress!!

File: src/webui/index.js [hunk 2]
```diff
@@ -3,15 +3,40 @@ import './utils/__setPublicPath__';
...
 import App from './app';
 
+// translated strings
+import localeData from "./i18n/locales.json";
+
+addLocaleData([...en,...es, ...pt]);
+
+// Define user's language. Different browsers have the user locale defined
+// on different fields on the `navigator` object, so we make sure to account
+// for these different by checking all of them
```
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply src/webui/index.js]: I am not done yet...I told you!

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (priscilawebdev) [reply package.json]: @juanpicado removed.

**Comment by juanpicado** [comment]
Any new source for translation has to be added to https://github.com/verdaccio/verdaccio/blob/master/crowdin.yaml#L7 

follow the same pattern there.

**Comment by priscilawebdev** [comment]
@juanpicado Let's talk about it tomorrow 😉

**Comment by juanpicado** [comment]
We decided to hold this ticket until `4.x` is being merged and test it properly.

**Pull request closed without merging**